schema/susemanager-schema-4.2.0-to-susemanager-schema-4.2.1/100-rpm.pkb.sql

```diff
@@ -0,0 +1,255 @@
+-- oracle equivalent source sha1 539cb03eb177b7e87992701071488bbb32bb0624
```

Comment:
Not sure if this should go into the uyuni release which is currently prepared.

If yes, please change the number of the script as we already have a `100-...` .
If not, you need to change the directory as it will be released for 4.2.1-to-4.2.2

Reply:
You are right - this time I deducted the right dir but I was not considering the release. Adjusting with the nextver dir.

Reply:
@mcalmer I'd say 4.2.0-to-4.2.1. The current spec on master is '4.2.0' and I still didn't tag for the Uyuni 2020.09 release.

What am I missing?

Reply:
Maybe you were thinking about the CVE? We'll not do a full release for that. Instead we'll release a set of packages to be installed on top of Uyuni 2020.07: https://github.com/uyuni-project/uyuni/wiki/Releasing-Uyuni-patches

This is nothing we are going to do frequently at all, but I created the mechanism for important situations such as this CVE.